.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: node_js
+
+node_js:
+    - '5.0.0'
+
+notifications:
+    email: false
+
+cache:
+  directories:
+    - $HOME/.npm
+
+sudo: false
```

Comment:
This is no longer needed: Travis defaults to the container setup unless told otherwise